rust_snuba/src/processors/functions.rs

```diff
@@ -15,35 +15,35 @@ pub fn process_message(
     let payload_bytes = payload.payload().context("Expected payload")?;
     let msg: InputMessage = serde_json::from_slice(payload_bytes)?;
 
-    let functions = msg.functions.iter().map(|from| {
-        Function {
-            profile_id: msg.profile_id,
-            project_id: msg.project_id,
+    let functions: Vec<Function> = msg
```

Comment:
these changes seem unrelated

Reply:
The idea was to use InsertBatch::from_rows everywhere as it's now the preferred method of constructing the InsertBatch